package.json

```diff
@@ -1,177 +1,181 @@
-        "@rollup/plugin-replace": "^3.0.1",
-        "@rollup/plugin-url": "^6.1.0",
-        "@semantic-release/changelog": "^5.0.1",
-        "@semantic-release/commit-analyzer": "^8.0.1",
-        "@storybook/addon-a11y": "^6.5.9",
-        "@storybook/addon-console": "^1.2.2",
-        "@storybook/addon-essentials": "^6.5.9",
-        "@storybook/addon-interactions": "^6.5.9",
-        "@storybook/react": "^6.5.9",
-        "@storybook/testing-library": "^0.0.9",
-        "@svgr/rollup": "^6.2.1",
-        "@svgr/webpack": "^5.5.0",
-        "@testing-library/jest-dom": "^5.11.8",
-        "@testing-library/react": "^11.2.2",
-        "@testing-library/react-hooks": "^5.1.0",
-        "@types/estree": "0.0.46",
-        "@types/jest": "^26.0.19",
-        "@types/react": "^17.0.1",
-        "@typescript-eslint/eslint-plugin": "^5.52.0",
-        "@typescript-eslint/parser": "^5.13.0",
-        "babel-jest": "^26.6.3",
-        "babel-loader": "^8.2.2",
-        "babel-preset-react-app": "^10.0.0",
-        "babylonjs-hook": "^0.1.1",
-        "body-parser": "^1.19.0",
-        "chromatic": "^5.6.2",
-        "colors": "^1.4.0",
-        "conventional-changelog-conventionalcommits": "^4.5.0",
-        "cors": "^2.8.5",
-        "cz-conventional-changelog": "^3.3.0",
-        "eslint": "^8.0.0",
-        "eslint-config-prettier": "^8.6.0",
-        "eslint-webpack-plugin": "^2.4.1",
-        "express": "^4.17.1",
-        "file-loader": "^6.2.0",
-        "http-proxy-middleware": "^1.0.6",
-        "identity-obj-proxy": "^3.0.0",
-        "jest": "^26.6.3",
-        "jest-canvas-mock": "^2.3.1",
-        "mkcert": "^1.5.0",
-        "mkdirp": "^1.0.4",
-        "postcss": "^8.2.8",
-        "postcss-loader": "^4.2.0",
-        "postcss-url": "^10.1.3",
-        "prettier": "2.2.1",
-        "react": "^17.0.1",
-        "react-babylonjs": "^3.0.15",
-        "react-dom": "^17.0.1",
-        "rollup": "^2.71.1",
-        "rollup-plugin-dts": "^4.1.0",
-        "rollup-plugin-peer-deps-external": "^2.2.4",
-        "rollup-plugin-postcss": "^4.0.2",
-        "rollup-plugin-rename-node-modules": "^1.2.0",
-        "rollup-plugin-typescript2": "^0.31.1",
-        "sass": "^1.69.7",
-        "sass-loader": "^10.1.0",
-        "seedrandom": "^3.0.5",
-        "semantic-release": "^17.3.7",
-        "shx": "^0.3.3",
-        "strip-comments": "^2.0.1",
-        "stylelint": "^13.8.0",
-        "stylelint-config-prettier": "^8.0.2",
-        "stylelint-config-recommended": "^3.0.0",
-        "stylelint-config-sass-guidelines": "^7.1.0",
-        "ts-jest": "^26.4.4",
-        "tslib": "^2.1.0",
-        "typescript": "^4.1.3"
-    },
-    "peerDependencies": {
-        "react": ">=16.8.0",
-        "react-dom": ">=16.8.0"
-    },
-    "overrides": {
-        "minimist@1.2.5": "1.2.6"
-    },
-    "author": "",
-    "license": "ISC"
-}
\ No newline at end of file
+  "name": "@microsoft/iot-cardboard-js",
```

Comment:
can we revert the formatting change so that we can see the actual diff?

Reply:
Done